package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "khi-devtools",
+  "version": "1.0.0",
+  "description": "Development tools for KHI project",
+  "private": true,
+  "devDependencies": {
+    "markdownlint-cli": "^0.37.0"
```

Comment:
Please use `markdownlint-cli2` instead of `markdownlint-cli`.

We use a GitHub Action of `markdownlint-cli2`, and the owner is the same engineer of markdownlint. this one sounds better.